### PR TITLE
test: add ramda placeholder and make tests consistent

### PR DESCRIPTION
**Pull request by js636f**
Batch 16

Files:

test/
- liftF.js
- liftFN.js
- list.js

src/
- liftF.js

### CODE REVIEW

**Comment by js636f** [comment]
@char0n, I edited some descriptions in the test/list.js file, but I'm not sure about currying. It seems to me this is a very special function because it supports unlimited list of arguments and returns something useful even when called without any arguments. I don't know how to curry it properly. If it have to be curried anyway - give me a hint please.

File: test/liftFN.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ describe('liftFN', function() {
     );
   });
 
-  it('retain order of arguments', function() {
+  it('retains order of arguments', function() {
```
Comment (char0n) [inline test/liftFN.js]: should retain order of arguments

File: test/list.js [hunk 2]
```diff
@@ -3,19 +3,19 @@ import { assert } from 'chai';
 import * as RA from '../src';
 
 describe('list', function() {
-  it('tests creating list from items', function() {
+  it('creates list from items', function() {
```
Comment (char0n) [inline test/list.js]: For this whole file here, we have a convention to always start a test description with a `should` word.

`creates list from another list` -> `should create list from another list`

File: test/liftFN.js [hunk 3]
```diff
@@ -26,7 +26,7 @@ describe('liftFN', function() {
     );
   });
 
-  it('throws error on variadic function is more arguments than arity', function() {
+  it('should throw an error on variadic function which have more arguments than its arity', function() {
```
Comment (char0n) [inline test/liftFN.js]: can we pls break this into 
```js
context('given variadic function with more arguments than its arity', function() {
  specify('should trow an error', function() {
   ...
  });
});

**Pull request merged**